helix-rest/src/main/java/org/apache/helix/rest/metadatastore/accessor/ZkRoutingDataReader.java

```diff
@@ -36,8 +36,8 @@
 import org.apache.helix.zookeeper.zkclient.exception.ZkNoNodeException;
 import org.apache.zookeeper.Watcher;
 
-
-public class ZkRoutingDataReader implements MetadataStoreRoutingDataReader, IZkDataListener, IZkChildListener, IZkStateListener {
+public class ZkRoutingDataReader
+    implements MetadataStoreRoutingDataReader, IZkDataListener, IZkChildListener, IZkStateListener {
```

Comment:
Has this been formatted properly?

Reply:
The previous formatting was not done by this PR. This is the result of using helix-style.xml to auto-format. 
Note: the previous formatting violates the first element in formatting guideline: the length exceeds 100 characters. (https://cwiki.apache.org/confluence/display/HELIX/Coding+Style)

Reply:
Turns out that the onboarding doc links to an incorrect coding style page. Correcting it.